032-MLOpsFromScratch/Student/04-ReleasePipeline.md

```diff
@@ -0,0 +1,67 @@
+# Challenge 4 – Create a Release Pipeline In Azure DevOps
+
+[< Previous Challenge](./03-BuildPipeline.md) - **[Home](../README.md)** - [Next Challenge >](./05-RetrainingAndEvaluation.md)
+
+
+## Introduction
+
+The Release pipeline demonstrates the automation of various stages/tasks
+involved in deploying an ML model and operationalizing the model in production.
+The stages generally constitute collecting the Build Artifacts, creating a web
+service and testing the web service. This web service that gets created in the
+Release Pipeline is a REST endpoint (a Scoring URI) used to predict/forecast on
+a new dataset. Additionally, it can be plugged into business applications to
+leverage the intelligence of the model.
+
+There are several ways to create a Release pipeline. The two most common and popular
+ways are: 
+
+-   using a YAML file that represents the entire pipeline,
+
+-   using an empty job and adding tasks sequentially
```

Comment:
Same comment here re capitalization and ending punctuation

Reply:
Done